db/seeds.rb

```diff
@@ -3,14 +3,47 @@
 #
 # Examples:
 #
-  videos = Video.create([
-    { title: 'Birdman', 
-      description: 'A washed-up actor, who once played an iconic superhero, battles his ego and attempts to recover his family, his career and himself in the days leading up to the opening of his Broadway play.',
-      small_cover_url: 'tmp/birdman.jpg',
-      large_cover_url: '',
-      category_id: 1 }])
 
-  Category.create(name: 'drama')
-  Category.create(name: 'comedy')
-  Category.create(name: 'action')
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Category.create([ {name: 'drama'}, 
+                  {name: 'comedy'}, 
+                  {name: 'action'}])
+
+User.create([
```

Comment:
`create!` is preferred in seeds as it raises an exception on invalid record and alerts you.